source/audio/appsVolume.py

```diff
@@ -105,24 +105,30 @@ def _updateAppsVolumeImpl(
 	_activeCallback.register()
 
 
+WASAPI_DISABLED_MESSAGE: str = _(
+	# Translators: error message when wasapi is turned off.
+	"Application volumes cannot be modified by NVDA when WASAPI is disabled. "
```

Comment:
I don't think "volumes" makes sense here. typo?

```suggestion
	"Application volume cannot be modified by NVDA when WASAPI is disabled. "
```